api/bonebudget/serializers.py

```diff
@@ -0,0 +1,11 @@
+from rest_framework import serializers
+from .models import Transaction
+
+
+class TransactionSerializer(serializers.ModelSerializer):
```

Comment:
How does the `rest_framework` `ModelSerializer` work? Does that produce JSON?

Reply:
That's right @reedlauber

> To convert the Model object to an API-appropriate format like JSON, Django REST framework uses the ModelSerializer class to convert any model to serialized JSON objects

Reply:
I didn't do a particularly deep dive on the how-it-works part, but there's more info here at the official docs: https://www.django-rest-framework.org/api-guide/serializers/

Reply:
Okay cool. Hopefully it returns a dictionary of some kind. Probably makes sense to use `JsonResponse` instead of `Response` below so that it sets the Content-type to `application/json`.

Reply:
Right! Thanks for pointing that out. I had to read some docs them, but it does appear JsonResponse is the way to go. I will remove unused imports as well 👍🏾

Reply:
Well, a current issue I'm faced with is the `AdminRenderer` the gui doesn't work with JsonResponse for some reason. I haven't been able to find a solution to this yet.

https://www.django-rest-framework.org/api-guide/renderers/#adminrenderer

Reply:
Wait, Reed I don't understand how exactly but it looks like `Response()` is delivering `Content-Type: Application/Json`. 

Let's maybe meet up at 6 tomorrow before Beers?


<img width="1280" alt="content-type" src="https://github.com/boneburrito/budgetsimple/assets/10481217/ddffb9e7-e609-42a0-876f-0013d6e41fad">